chcemvediet/apps/inforequests/admin.py

```diff
@@ -315,3 +317,58 @@ def get_queryset(self, request):
         queryset = queryset.select_related(u'branch')
         queryset = queryset.select_related(u'email')
         return queryset
+
+    def get_actions(self, request):
+        actions = super(ActionAdmin, self).get_actions(request)
+        if u'delete_selected' in actions:
+            del actions[u'delete_selected']
+        return actions
+
+    def has_delete_permission(self, request, obj=None):
+        if obj is None:
+            return True
+        if obj.type in [Action.TYPES.REQUEST, Action.TYPES.ADVANCED_REQUEST]:
+            return False
+        if len(obj.branch.actions) > 1:
+            return True
+        return False
+
+    def render_delete_form(self, request, context):
+        using = router.db_for_write(self.model)
+        collector = admin.utils.NestedObjects(using)
+        collector.collect([context[u'object']])
+        to_delete = collector.nested()
+        for object in traverse(to_delete):
```

Comment:
Nezvykneme pouzivat premennu s nazvom `object` lebo je to globalna funkcia. Preto zvycajne pouzivame iba `obj`.

Reply:
`obj` ako nazov nemozeme pouzit, pretoze premennu `obj` pouziva parent metoda `delete_object`

Reply:
:ok: